client/checkout/platform-checkout/express-button/index.js

```diff
@@ -0,0 +1,35 @@
+/* global jQuery */
+/**
+ * External dependencies
+ */
+import ReactDOM from 'react-dom';
+
+/**
+ * Internal dependencies
+ */
+import { WoopayExpressCheckoutButton } from './woopay-express-checkout-button';
+
+const renderPlatformCheckoutExpressButton = () => {
+	const platformCheckoutContainer = document.getElementById(
+		'wcpay-platform-checkout-button'
+	);
+
+	ReactDOM.render(
```

Comment:
How about checking if `platformCheckoutContainer` exists before trying to render the react component?

Reply:
Resolved in fe01b1ee